Sources/CoreDataCandy/Database/DatabaseModel/DatabaseModel+FieldInterfaceProtocol.swift

```diff
@@ -17,3 +17,21 @@ public extension DatabaseModel {
         self[keyPath: keyPath].currentValue(in: entity)
     }
 }
+
+public extension Collection where Element: DatabaseModel {
+
+    /// Map the current value of the field
+    func mapCurrent<F: FieldInterfaceProtocol>(_ keyPath: KeyPath<Element, F>) -> [F.Value] where Element.Entity == F.Entity {
```

Comment:
Could be useful to have it for Children & Parent too

Reply:
That's the case for the `Children` already (I added it in the last commit). For the parent, as it's a single model I don't think it would make sense.